examples/web-wagmi/tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "ESNext",
     "useDefineForClassFields": true,
     "lib": ["DOM", "DOM.Iterable", "ESNext"],
-    "allowJs": false,
+    "allowJs": true,
```

Comment:
Is this intentional?

Reply:
All changes to the `package.json` in `src/packages` are intentional, this one is just making everything in sync across all `tsconfig`.